javascript/extractor/src/com/semmle/js/extractor/Main.java

```diff
@@ -134,7 +135,8 @@ public void run(String[] args) {
 
     TypeScriptParser tsParser = extractorState.getTypeScriptParser();
     tsParser.setTypescriptRam(extractorConfig.getTypeScriptRam());
-    if (containsTypeScriptFiles()) {
+    boolean containsTypeScriptFiles = containsTypeScriptFiles();
```

Comment:
is this change really required?

Reply:
Ah, you're right it isn't. Originally I only wanted to pay the I/O cost if there were any TypeScript files, but that doesn't work in QLTest where there is one extractor invocation per `.js` file.